pvacseq/lib/output_parser.py

```diff
@@ -514,7 +467,110 @@ def execute(self):
                     row['Tumor RNA Depth'] = tsv_entry['trna_depth']
                 if 'trna_vaf' in tsv_entry:
                     row['Tumor RNA VAF'] = tsv_entry['trna_vaf']
+                if 'fusion_position' in tsv_entry:
+                    row['Fusion Position'] = tsv_entry['fusion_position']
+                else:
+                    row['Fusion Position'] = 'NA'
                 tsv_writer.writerow(row)
 
         tmp_output_filehandle.close()
         os.replace(tmp_output_file, self.output_file)
+
+class DefaultOutputParser(OutputParser):
```

Comment:
Mh, good question. These are the outputs from processing a VCF. I'm hesitant to call it a `VcfOutputParser` because there are intermediate steps that produce different formats. Maybe `NonsynonymousVariantOutputParser` which would mirror the `FusionOutputParser` . But that seems clunky as well. @chrisamiller any opinion for a better name for this parser?